main/src/library/MutMap.flix

```diff
@@ -21,6 +21,23 @@ pub enum MutMap[k: Type, v: Type, r: Region] {
     case MutMap(Region[r], Ref[Map[k, v], r])
 }
 
+instance Index[MutMap[k, v, r]] with Order[k] {
+    type Idx = k
+    type Elm = v
+    type Aef = r + InvalidKey
+    pub def get(i: k, t: MutMap[k, v, r]): v \ r + InvalidKey = {
```

Comment:
I think a linebreak after associated types/effects is reasonable?

Reply:
I did it now but I'm not a fan